frontend/src/app/main/ui/workspace/sidebar/assets.cljs

```diff
@@ -697,8 +698,7 @@
         :open? true
         :filters @filters}]
 
-      (for [file (->> (vals libraries)
-                      (sort-by #(str/lower (:name %))))]
+      (for [file (sort-by #(str/lower (:name %)) libraries)]
```

Comment:
Same here. You undid the threading macro but I think the problem is the anonymous function

Reply:
Done. Removed the thread because I don't need (vals) here, but it's still useful for the sort.